setup.py

```diff
@@ -824,7 +824,16 @@ def make_relative_rpath_args(path):
                   include_dirs=[],
                   library_dirs=library_dirs,
                   extra_link_args=extra_link_args + main_link_args + make_relative_rpath_args('lib'))
+    C_flatbuffer = Extension("torch._C_flatbuffer",
```

Comment:
It's a pretty high-level and user-facing extension. Why not adding `_flatbuffer` to `torch._C', so we don't have to add a new extension here?

Reply:
`//caffe2:torch` cannot depend on flatbuffer without breaking some downstreams. So the use of flatbuffer features for them is opt-in. Therefore, it has to be separate .so to be generated, and thus different names.